assets/data/cards_pt-br.json

```
@@ -95,6 +95,10 @@
         {
             "title": "Frameworks",
             "description": "São como coleções de ferramentas, componentes e soluções que você pode encontrar em um mesmo pacote (o framework) que facilitam a criação de aplicações complexas."
+        },
+        {
+            "title": "API Rest",
+            "description": "Uma API Rest, e uma API(Application Programming Interface) que segue um conjunto de regras, convenções e protocolos HTTP, que permite que aplicativos, sites se comuniquem pela internet.",
```

O que você acha dessa sugestão? Modifiquei levemente e removi a vírgula no final da frase do `description` pois ela não é necessária.

```suggestion
            "description": "Uma API Rest é uma API (Application Programming Interface ou Interface de Programação de Aplicativos) que segue um conjunto de regras, convenções e protocolos HTTP, possibilitando a comunicação entre aplicativos e sites pela internet."
```

---

Eu achei que ficou muito melhor assim rsrsrs

---

Eu devo fazer a alteração e subir novamente a descrição?

---

@franSborges pode deixar que eu ajusto e já subo :)